webapp/components/Paginate/Paginate.spec.js

```diff
@@ -1,39 +1,35 @@
 import { mount } from '@vue/test-utils'
 
-import Paginate from './Paginate'
+import PaginationButtons from './PaginationButtons'
 
 const localVue = global.localVue
 
-describe('Paginate.vue', () => {
-  let propsData, wrapper, nextButton, backButton
-
-  beforeEach(() => {
-    propsData = {}
-  })
+describe('PaginationButtons.vue', () => {
+  let propsData = {}
+  let wrapper
+  let nextButton
+  let backButton
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T15:59:24Z" title="Wednesday, January 15th 2020, 4:59:24 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,35 +5,31 @@ import Paginate from './Paginate'
 const localVue = global.localVue
 
 describe('Paginate.vue', () => {
-  let propsData, wrapper, nextButton, backButton
-
-  beforeEach(() => {
-    propsData = {}
-  })
+  let propsData = {}
+  let wrapper
+  let nextButton
+  let backButton
```


no really a fan of this change, but it's not really a big deal.
I don't see a justification to add extra `let` statements, and lines of code.

Also setting `propsData` to an initial value of an empty object doesn't have the same effect as resetting it to an empty object before every test. I feel like resetting the value is a safer practice so that we don't get any false negative where we think we are testing one thing, but we are not cause `propsData` is leaking from one it block to another...

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-22T07:38:22Z" title="Wednesday, January 22nd 2020, 8:38:22 am +01:00">Jan 22, 2020</time>_

---


You're right! I didn't think about it – just thought I could make it shorter by initializing the value. The extra lines were then added by `prettier`, I didn't even notice...